### ISSUE TITLE
[focusgroup] should there be a `no-roving` token to disable focusgroup's adjustment to sequential focus navigation?

### ISSUE THREAD

**Issue by janewman** [openui/open-ui#1302]
@gfellerph suggested this in https://github.com/openui/open-ui/pull/1298#discussion_r2414953049

With this attribute, any focusable element that doesn't have a negative tabindex would appear in tab order, with focusgroup **not** making any adjustments to sequential focus navigation.

This means f a user only used tab/shift+tab, it would be as if focusgroup was not applied, but they could still move focus between items with the arrow keys.

Personally, I feel like this should be considered out of scope but would be convinced if authors feel like this gap is what is stopping them from being able to adopt the feature.

**Comment by scottaohara** [comment]
this does feel like scope creep, and what would the benefit be to the user if tab and arrow keys both navigated to the same elements?

generally, when encountering that behavior in an interface, it's a bug.

**Comment by gfellerph** [comment]
That's reasonable, I think this idea can be abandoned unless a good reason to add it comes up. I'll close the issue, please re-open if that was too early.
